dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/codec/NettyDecoder.java

```diff
@@ -55,15 +55,18 @@ protected void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) t
             case MAGIC:
                 checkMagic(in.readByte());
                 checkpoint(State.COMMAND);
+                break;
             case COMMAND:
                 commandHeader.setType(in.readByte());
                 checkpoint(State.OPAQUE);
+                break;
             case OPAQUE:
                 commandHeader.setOpaque(in.readLong());
                 checkpoint(State.BODY_LENGTH);
             case BODY_LENGTH:
                 commandHeader.setBodyLength(in.readInt());
                 checkpoint(State.BODY);
+                break;
```

Comment:
You cannot directly break here, please refer to #3035

Reply:
@gabrywu gabrywu gave some suggestions, maybe you can complete it. You can refer to https://github.com/apache/incubator-dolphinscheduler/pull/2969

Reply:
> You cannot directly break here, please refer to #3035

I know the reason for removing break. Should I add a break at the last case? If don't do this, there will be many WARN logs that shouldn't appear in the log.

Reply:
Of course, you can add a break at the last case